docker/cygnus-ngsi/Dockerfile

```diff
@@ -32,6 +32,7 @@ ENV CYGNUS_LOG_APPENDER "console"
 ENV CYGNUS_SERVICE_PORT "5050"
```

Comment:
Please, add a line to CHANGES_NEXT_RELEASE file briefly described the changes include in this PR. For instance:

```
[cygnus-ngsi][Docker] Add new env var  CYGNUS_MONITORING_TYPE
```